app/models/sources/fog.rb

```diff
@@ -64,32 +64,45 @@ def decommission_proxy(proxy)
     # Adds them to the site if provided
     # Parameters:
     #   site - What site to add the found servers to (if any)
-    def find_orphaned_servers!(*args)
-      num_servers_found = 0
+    #   desired_proxy_count - Until the site owns this many proxies, add new
+    #     proxies to site
+    def find_orphaned_servers!(
+      site: NoSite,
+      desired_proxy_count: self.max_proxies
+    )
```

Comment:
I like the removal of the `*args`, but can this declaration be placed on one line instead of spread out across 4? It's not that complex.